crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -0,0 +1,201 @@
+//! Utilities for working with events to decide whether they are suitable for
+//! use as a RoomInfo::latest_event
```

Comment:
```suggestion
//! use as a [RoomInfo::latest_event].
```

so that we have a link to this property, plus, if the property gets renamed or is removed, the doc will break and we will know it's outdated.